src/lib.rs

```diff
@@ -297,6 +297,68 @@ pub fn fbm_simplex_3d(pos: Vec3, octaves: usize, lacunarity: f32, gain: f32) ->
     sum
 }
 
+/// Also called Voronoi or cellular noise
+pub fn worley_noise_2d(pos: Vec2) -> Vec2 {
+    const K: f32 = 1.0 / 7.0;
+    const KO: f32 = 3.0 / 7.0;
+    let jitter = 1.0;
```

Comment:
I'm not that famililar with worley noise. Would having jitter as an input be useful?